### ISSUE TITLE
Can not install stickers

### ISSUE THREAD

**Issue by caybro** [status-im/status-desktop#10034]
... and no error message shown to the user:

![image](https://user-images.githubusercontent.com/5377645/227933484-beb5b41c-9fde-41a7-b22c-3ef8e526a5e7.png)

Error in console:
```
DBG 2023-03-27 13:50:38.760+02:00 [threadpool task thread] initiating task   topics="task-threadpool" tid=24218 file=threadpool.nim:48 messageType=InstallStickerPackTaskArg:ObjectType threadid=24218 task="{\"$type\":\"InstallStickerPackTaskArg:ObjectType\",\"packId\":\"15\",\"chainId\":1,\"hasKey\":true,\"vptr\":44762016,\"slot\":\"onStickerPackInstalled\",\"tptr\":15050752}"
DBG 2023-03-27 13:50:38.760+02:00 NewBE_callPrivateRPC                       topics="rpc" tid=24218 file=core.nim:27 rpc_method=stickers_install
ERR 2023-03-27 13:50:38.870+02:00 rpc response error                         topics="rpc" tid=24218 file=core.nim:36 err="\nstatus-go error [methodName:stickers_install, code:-32000, message:fallback failed with \'429 Too Many Requests: {\"jsonrpc\":\"2.0\",\"id\":8157,\"error\":{\"code\":-32005,\"message\":\"daily request count exceeded, request rate limited\",\"data\":{\"rate\":{\"allowed_rps\":1,\"backoff_seconds\":30,\"current_rps\":3},\"see\":\"https://infura.io/dashboard\"}}}\'. run error was \'hystrix: circuit open\' ]\n"
ERR 2023-03-27 13:50:38.870+02:00 error doing rpc request                    topics="rpc" tid=24218 file=core.nim:40 methodName=stickers_install exception="\nstatus-go error [methodName:stickers_install, code:-32000, message:fallback failed with \'429 Too Many Requests: {\"jsonrpc\":\"2.0\",\"id\":8157,\"error\":{\"code\":-32005,\"message\":\"daily request count exceeded, request rate limited\",\"data\":{\"rate\":{\"allowed_rps\":1,\"backoff_seconds\":30,\"current_rps\":3},\"see\":\"https://infura.io/dashboard\"}}}\'. run error was \'hystrix: circuit open\' ]\n"
ERR 2023-03-27 13:50:38.871+02:00 Error installing stickers                  tid=24218 file=async_tasks.nim:83 message="\nstatus-go error [methodName:stickers_install, code:-32000, message:fallback failed with \'429 Too Many Requests: {\"jsonrpc\":\"2.0\",\"id\":8157,\"error\":{\"code\":-32005,\"message\":\"daily request count exceeded, request rate limited\",\"data\":{\"rate\":{\"allowed_rps\":1,\"backoff_seconds\":30,\"current_rps\":3},\"see\":\"https://infura.io/dashboard\"}}}\'. run error was \'hystrix: circuit open\' ]\n"
```

## Steps to reproduce

Click one of the "Free" sets in the stickers popup

### Additional Information

- Status desktop version: master
- Operating System: linux

**Comment by alaibe** [comment]
This error is about the number of request made, this happens when we hit the rate limit. I am not sure it is an RC as it didn't change since last time, this has always been that way, also it quite unlikely it happens in production due to better key limit

Probably not that urgent and should be wallet-team?

**Comment by jrainville** [comment]
Ah, my bad, I saw the sticker issue, so I assumed it was just something related the chat, but ok, I'll put back the wlalet-team label.

Up to you to move it to 0.12 or close it

**Comment by alaibe** [comment]
@caybro by default: we all use a very limited key on development (the one in the makefile), it is a free version, so now that the wallet is enabled by default, much more request are being made and as such cause this issue.
In release: we inject key that have much more resources available.

@jrainville we are working on error state with blockchain, that could be part of it

**Comment by caybro** [comment]
> @caybro by default: we all use a very limited key on development (the one in the makefile), it is a free version, so now that the wallet is enabled by default, much more request are being made and as such cause this issue. In release: we inject key that have much more resources available.
> 
> @jrainville we are working on error state with blockchain, that could be part of it

OK understood.... 99% of users will be very confused though why a messanger/chat doesn't work because of some hidden feature related to wallet. Also the missing error message doesn't help the situation much :)

**Comment by alaibe** [comment]
The error message is being worked on. However we expect this error to not happen with a release build, you have the privilege to use a dev build with a nearly unusable api key :)

I would even advise to go on poket/infura, register and get your own key. (this is what we all do in wallet team and i am slowly trying to convince everyone this is the way to go)

**Comment by Khushboo-dev-cpp** [comment]
@John-44 @benjthayer We have another use case for stickers in case it is not possible to install because of site connection errors. We will be showing a toast saying connection is not there, but on the screen itself we have no way of showing this erro.

**Comment by caybro** [comment]
> should this be closed? @caybro @alaibe

Nope, I guess we should still display some error message if the installation cannot be carried on for whatever reason

**Comment by anastasiyaig** [comment]
okay handling the edge case of installation failure will be tracked in separate task (i assume it needs designs too)
